spring/src/main/java/org/axonframework/spring/config/ConfigurerFactoryBean.java

```diff
@@ -41,6 +41,7 @@
  * @author Allard Buijze
  * @since 4.2
  */
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.